bin/start.py

```diff
@@ -3,6 +3,7 @@
 
 
 def start_smoderp2d():
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
What is the purpose of this change? How is it connected to the implicit solution?

Reply:
OK. As it does not break anything but fixes it on your laptop, this line will be kept in the code unless it causes some troubles to anybody (although it should probably be solved by setting `PYTHONPATH`).